index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h2 class="header-welcome">Welcome</h2>
+            <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="menu icon" />
+        </nav>
+    </header>
+    <main>
+        <section class="main-page">
+            <p class="introduction-text"><span class="dark-blue">Hey there. I'm <br>Eden<br></span>
+            <span class="green-content">I'm a software developer</span></p>
+            <p class="description-profession">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hestiate to contact me.</p>
+
+            <aside class="social-media">
+                <img src="./assets/social-media-icons/twitter.svg" alt="twitter icon"/>
+                <img src="./assets/social-media-icons/Linkedin.svg" alt="linkedin icon"/>
+                <img src="./assets/social-media-icons/medium.svg" alt="medium icon"/>
+                <img src="./assets/social-media-icons/github.svg" alt="github icon"/>
+                <img src="./assets/social-media-icons/angellist.svg" alt="angellist icon"/>
+            </aside>
```

Comment:
Good job adding the `src` and the `alt` attributes to the `img` tag 🥇 

The `aside` tag defines some content aside from the content it is placed in. [Check this link](https://www.w3schools.com/tags/tag_aside.asp)

- [ ] Actually, this is a list of different items. In this case, it's apply the best practices of the `ul` tag, and inside them, you can create an `li` with an `a` tag for each `img` that you have, so each icon will have its corresponding `href` to the differents social icons.

Kindly make these changes so your different icons can give to the viewer an interactive visit to your social places.